maqs-webservices/src/test/java/com/cognizantsoftvision/maqs/webservices/MediaTypeUnitTest.java

```diff
@@ -4,14 +4,13 @@
 
 package com.cognizantsoftvision.maqs.webservices;
 
-import com.cognizantsoftvision.maqs.base.BaseGenericTest;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
 /**
  * The Media Type unit test class.
  */
-public class MediaTypeUnitTest extends BaseGenericTest {
+public class MediaTypeUnitTest extends BaseWebServiceTest {
```

Comment:
Why are we implementing the base service interface if it is not required for the tests?